Makefile

```diff
@@ -42,7 +42,8 @@ lint:
 test: FLAGS ?=
 test: $(BIN)/testtime
 	FIRESTORE_EMULATOR_HOST="$(firestore_emulator)" \
-	$(GO_ENV) $(GO) test $(FLAGS) -race -overlay="$(shell $(BIN)/testtime -u)" ./...
+	GOOGLE_CLOUD_PROJECT="$(default_project)" \
+	$(GO_ENV) $(GO) test $(FLAGS) -trimpath -race -overlay="$(shell $(BIN)/testtime -u)" ./...
```

Comment:
> -trimpath
	remove all file system paths from the resulting executable.
	Instead of absolute file system paths, the recorded file names
	will begin either a module path@version (when using modules),
	or a plain import path (when using the standard library, or GOPATH).

https://pkg.go.dev/cmd/go#hdr-Compile_packages_and_dependencies